### PR TITLE
Add type associations to schema

### PR DESCRIPTION
**Pull request by ryanahamilton**
This will allow us to be more efficient on the client side where multiple queries could be combined and also let us factor out fields like `deploymentCount` in the future.

### CODE REVIEW

File: src/schema.graphql [hunk 1]
```diff
@@ -129,6 +129,8 @@ type Workspace {
   properties: JSON
   label: String
   users: [User]
+  deployments: [Deployment]
+  serviceAccounts: [ServiceAccount]
```
Comment (schnie) [inline src/schema.graphql:133]: @ryanahamilton do these work without defining the backing resolvers?

Comment (ryanahamilton) [reply src/schema.graphql:133]: Yeah, they appear to Just Work™ in my local testing of the new query I added in astronomer/astro-ui#484.

Comment (ryanahamilton) [reply src/schema.graphql:133]: I take that back… it isn't scoping to only "active".

Comment (ryanahamilton) [reply src/schema.graphql:133]: Updated with scoped resolver and tests.

**Pull request merged**